Kickstarter-iOS/Views/Cells/SettingsNewslettersCell.swift

```diff
@@ -20,19 +20,28 @@ internal final class SettingsNewslettersCell: UITableViewCell, ValueCell {
 
   public weak var delegate: SettingsNewslettersCellDelegate?
 
-  func configureWith(value: (newsletter: Newsletter, user: User)) {
+  override func awakeFromNib() {
+    super.awakeFromNib()
+
+    _ = self
+      |> \.accessibilityElements .~ [self.newslettersSwitch]
+  }
 
+  func configureWith(value: (newsletter: Newsletter, user: User)) {
     self.viewModel.inputs.configureWith(value: value)
 
+    _ = self.newslettersSwitch
+      |> \.accessibilityLabel %~ { _ in value.newsletter.displayableName }
```

Comment:
This string does not look like it's being evaluated for localization so we don't need to use %~.
`|> \.accessibilityLabel .~ value.newsletter.displayableName` should be enough.

Reply:
This sounds correct to me - my understanding of `.~` vs `%~` is that in the latter the current value is passed in to a function so that one might return a value based on the current. Generally, the way that we use it we tend to ignore the current value but we use `%~` because we can pass it a closure that is evaluated at runtime. We tend to use this for `Strings.*` functions because under the hood those are calling `localizedString()` which means that if we change the language on our `Environment` between calls to `bindStyles()` we will get the updated translations each time as opposed to the original function being captured and executed each time.

In the case of setting this value from another variable as in the case above, it's up to the place where that variable is initially set in order for the localization to take place.

^ all of this is just based on my understanding of how it works, tbh I'm not sure if that's completely correct. I think that a way to test this would be with snapshot tests. I will try to find an example.